src/System.CommandLine/RootCommand.cs

```diff
@@ -42,8 +52,14 @@ public override string Name
             Assembly.GetEntryAssembly() ??
             Assembly.GetExecutingAssembly();
 
+        /// <summary>
+        /// The name of the executable currently running, or if missing the first string argument.
```

Comment:
> or if missing the first string argument.

I'm not sure I understand this bit.

Reply:
Looking at the code, that's what I thought it was doing.